content/en/continuous_integration/setup_tests/java.md

```diff
@@ -37,9 +25,9 @@ Supported CI providers:
 
 ### Maven
```

Comment:
Would it be possible to add ![Maven Central](https://img.shields.io/maven-central/v/com.datadoghq/dd-java-agent?style=plastic) to the docs or would that not be allowed because it uses an external service? It allows users to know what the latest version is without navigating away from the docs page, which is nice.

Reply:
I'll add it and see what the docs team says on the review